inc/namespace.php

```diff
@@ -352,10 +352,11 @@ function ( $error ) {
 	];
 
 	$metadata = [
-		'$_GET'     => $_GET,
-		'$_POST'    => $_POST,
-		'$_COOKIE'  => $_COOKIE,
-		'$_SERVER'  => $_SERVER,
+		'$_GET'        => $_GET,
+		'$_POST'       => $_POST,
+		'$_COOKIE'     => $_COOKIE,
+		'$_SERVER'     => $_SERVER,
+		'sent_headers' => headers_list(),
```

Comment:
btw, I'd rather we have more structured data, like `metadata.response.headers`

Reply:
`http_response_code()` already has a place in the "native" response data, there's just no place for headers in there.

Reply:
Right; I'd still have something more structured than `sent_headers`. I'd like us to possibly eliminate the raw variables at some point.